README.md

```diff
@@ -1490,21 +1490,17 @@ Doesn't support static type inference 😕 Immediate disqualification, sorry Joi
 
 [https://github.com/jquense/yup](https://github.com/jquense/yup)
 
-Yup is a full-featured library that was implemented first in vanilla JS, with TypeScript typings added later.
+Yup is a full-featured library that was implemented first in vanilla JS, and later rewritten in TypeScript.
 
 Differences
 
 - Supports for casting and transformation
 - All object fields are optional by default
-- Non-standard `.required()`¹
-- Missing object methods: (pick, omit, partial, deepPartial, merge, extend)
```

Comment:
I also removed `merge` and `extend` here because yup has always had the equivalent of this, `concat()` and `shape()`, obviously the implementation is a bit different but conceptually they are the same methods.